[DOCS] Reason statement shown in alert rendered view (#2540)
elastic/security-docs: docs/detections/alerts-view-details.asciidoc

* *Summary*: Displays general details such as the alert's status, severity, risk score, and a link to the detection rule that produced the alert.

> * *Alert-rendered reason statement*: Summarizes alert details in a human-readable format. Shows high-level details, including the alert severity (`kibana.alert.severity`) and the rule that generated the alert (`kibana.alert.rule.name`). Fields are interactive, hover over one to access the available actions.

Author: @joepeeples this is a fair point. What might be "readable" to one group could be perceived as "non-readable" by others. Switching to "contextual format" could work, but I still feel as though that makes some assumptions, like the user already has the context necessary to understand how they should read the statement -- and some might, but that could be me treating our users as a monolithic group, which they aren't.

@andrew-goldstein @joepeeples do you think "logically-arranged" would work? Maybe something like:

```suggestion
* *Alert-rendered reason statement*: Provides alert details in a logically-arranged format. Shows high-level details, including the alert severity (`kibana.alert.severity`) and the rule that generated the alert (`kibana.alert.rule.name`). Fields are interactive, hover over one to access the available actions.
```

> * *Alert-rendered reason statement*: Summarizes alert details in a human-readable format. Shows high-level details, including the alert severity (`kibana.alert.severity`) and the rule that generated the alert (`kibana.alert.rule.name`). Fields are interactive, hover over one to access the available actions.

Reviewer: > Switching to "contextual format" could work, but I still feel as though that makes some assumptions, like the user already has the context necessary to understand how they should read the statement 

I meant more that the format establishes the context between the fields, but you're right, this could be easily misconstrued

> * *Alert rendered reason statement*: Summarizes the alert and renders the statement in a human-readable format. High-level alert details are provided, including the alert severity (`kibana.alert.severity`) and the rule that generated the alert (`kibana.alert.rule.name`). Alert details are interactive. Hover over them to access multiple actions.

Reviewer 2: Consider removing the text  ~and renders the statement~, shown as ~strikethrough~ below:

*Alert rendered reason statement*: Summarizes the alert ~and renders the statement~ in a human-readable format. High-level alert details are provided, including the alert severity (`kibana.alert.severity`) and the rule that generated the alert (`kibana.alert.rule.name`). Alert details are interactive. Hover over ~them~ to access multiple actions.

Also consider (in the text above) replacing ~them~ with _fields_, such that the new text reads:

Hover over ~them~ fields to access multiple actions.

> * *Alert-rendered reason statement*: Summarizes alert details and assembles the information into a human-readable format. High-level details are provided, including the alert severity (`kibana.alert.severity`) and the rule that generated the alert (`kibana.alert.rule.name`). Alert fields are interactive. Hover over alert fields to access multiple actions.

Author: Note to self: Need to refresh `alert-details-flyout.png` to show the updated reason statement and event renderer. Screenshot should also include the updated versions of the insights sections.

Author: Done

> * *Alert-rendered reason statement*: Summarizes alert details and assembles the information into a human-readable format. High-level details are provided, including the alert severity (`kibana.alert.severity`) and the rule that generated the alert (`kibana.alert.rule.name`). Alert fields are interactive. Hover over alert fields to access multiple actions.

Author: I like your revision, @benironside ! I just made some minor adjustments:

```suggestion
* *Alert-rendered reason statement*: Summarizes alert details in a human-readable format. Shows high-level details, including the alert severity (`kibana.alert.severity`) and the rule that generated the alert (`kibana.alert.rule.name`). Fields are interactive, hover over one to access the available actions.
```

> * *Event renderer*: Surfaces important alert details to provide useful context for understanding the alert. Alert details are assembled into a human-readable format and alert fields are interactive.

Reviewer 2: To illustrate the difference between the details in the _Alert-rendered reason statement_:

> **Alert-rendered reason statement**: Summarizes alert details

and the _Event renderer_ description:

> **Event renderer**: Surfaces important alert details

consider adding some examples to the Event renderer description, like the suggestion in _italics_ below:

**Event renderer**: Surfaces important alert details, _for example, file paths or process arguments_, to provide useful context for understanding the alert. Alert details are assembled into a human-readable format and alert fields are interactive.

Author: ```suggestion
* *Event renderer*: Surfaces important alert details, for example, file paths or process arguments, to provide useful context for understanding the alert. Alert details are assembled into a human-readable format and alert fields are interactive.
```

> * *Event rendered reason statement*: Surfaces important alert details to immediately provide useful context for understanding the alert. The statement is also rendered in a human-readable format and alert details are interactive.

Reviewer 2: Consider replacing ~Event rendered reason statement~ with "Event renderer", because the alert renderer (today) shares the same content as the `kibana.alert.reason` field, but the Event renderer is unrelated to the "reason" field.

> * *Event renderer*: Surfaces important alert details, for example, file paths or process arguments, to provide useful context for understanding the alert. Alert details are assembled into a human-readable format and alert fields are interactive.

Reviewer 3: ```suggestion
* *Event renderer*: Displays relevant event details, for example file paths or process arguments, to provide context for the alert. Shows details in a human-readable format. Fields are interactive, hover over one to access the available actions.
```

Author: ```suggestion
* *Event renderer*: Displays relevant event details, for example file paths or process arguments, to provide context for the alert. Shows alert details in a human-readable format. Fields are interactive, hover over one to access the available actions.
```

+

> NOTE: This statement only displays if an event renderer exists for the alert type.

Reviewer 2: Similar to the above, consider replacing ~This statement only displays~ with "An Event renderer is only displayed"
